### DISCUSSION TITLE
Формат даты из селекта #213

### DISCUSSION THREAD

**Discussion by Makhmut2445** [totumonline/totum-ru-issues-and-discussions#105] [Как это сделать?] [answered]
![image](https://user-images.githubusercontent.com/103171127/162692639-9510e4c0-a447-4187-960f-10e1d54a364b.png)
При выборе № договора, автоматически заполняются следующие столбцы с другой таблицы.
Таблица с которой подхватываются данные:
![image](https://user-images.githubusercontent.com/103171127/162692882-d0cdb27b-1702-4c05-bec8-56b5fe6762f7.png)
В таблице договоры дата в нормальном формате, при автозаполнении в другой таблице, формат меняется.. Можно ли взять отображение даты? А не основание... 
![image](https://user-images.githubusercontent.com/103171127/162697773-dbbc0477-c106-4c40-854e-684494803919.png)

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Есть вот такой раздел в учебном курсе: https://ru.docs.totum.online/date-processing

Из него вы узнаете, что даты хранятся в формате `Y-m-d` и `Y-m-d H:i` — поэтому когда вы берете значение из поля тип Дата в поле тип Строка у вас будет `Y-m-d`.

Либо Строку надо заменить на Дату, либо отформатировать вывод функцией `dateFormat` (второй вариант, только если дальше эта дата использоваться не будет, тк даты могут сравниваться только в `Y-m-d`).
